### ISSUE TITLE
serving部署seg，能不能让服务输出置信度呢

### ISSUE THREAD

**Issue by Enchanted0911** [PaddlePaddle/PaddleSeg#2883]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

我使用paddleserving服务化部署，但是输出好像只得到了mask，有没有办法能得到置信度信息呢

**Comment by juncaipeng** [comment]
使用export.py导出预测模型的时候，可以设置 `--output_op none`，这样得到的就是预测的logits，也可以设置 `--output_op softmax`。

文档：https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.7/docs/model_export_cn.md

![image](https://user-images.githubusercontent.com/52520497/210046598-456f6f8c-8f15-4a56-9165-3742cfe8ef98.png)

**Comment by Enchanted0911** [comment]
> 使用export.py导出预测模型的时候，可以设置 `--output_op none`，这样得到的就是预测的logits，也可以设置 `--output_op softmax`。
> 
> 文档：https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.7/docs/model_export_cn.md
> 
> ![image](https://user-images.githubusercontent.com/52520497/210046598-456f6f8c-8f15-4a56-9165-3742cfe8ef98.png)

好的，感谢！
